master/buildbot/changes/gitpoller.py

```diff
@@ -52,16 +57,14 @@ class GitPoller(base.ReconfigurablePollingChangeSource, StateMixin, GitMixin):
         "project",
         "pollAtLaunch",
         "buildPushesWithNoCommits",
-        "sshPrivateKey",
-        "sshHostKey",
-        "sshKnownHosts",
         "pollRandomDelayMin",
         "pollRandomDelayMax",
+        "_git_ssh_auth",
```

Comment:
Not sure if this plus the fact `AbstractGitSSHAuth` inherit from `ComparableMixin` will do what I expect.
Not sure how to test this, would welcome any pointers.

Reply:
ComparableMixin compares properties recursively so it does the right thing here.